docs/BR.md

```diff
@@ -2147,7 +2171,39 @@ The contactphone property MAY be critical if the domain owner does not want CAs
 
 ### B.2.1. DNS TXT Record Email Contact
 
-The DNS TXT record MUST be placed on the "_validation-contactemail" subdomain of the domain being validated.  The entire RDATA value of this TXT record MUST be a valid email address as defined in RFC 6532 section 3.2, with no additional padding or structure, or it cannot be used.
+The DNS TXT record MUST be placed on the "`_validation-contactemail`" subdomain of the domain being validated.  The entire RDATA value of this TXT record MUST be a valid email address as defined in RFC 6532 section 3.2, with no additional padding or structure, or it cannot be used.
 
 ### B.2.2. DNS TXT Record Phone Contact
-The DNS TXT record MUST be placed on the "_validation-contactphone" subdomain of the domain being validated.  The entire RDATA value of this TXT record MUST be a valid Global Number as defined in RFC 3966 section 5.1.4, or it cannot be used.
+The DNS TXT record MUST be placed on the "`_validation-contactphone`" subdomain of the domain being validated.  The entire RDATA value of this TXT record MUST be a valid Global Number as defined in RFC 3966 section 5.1.4, or it cannot be used.
+
+# APPENDIX C – Issuance of Certificates for .onion Domain Names
+This appendix defines permissible verification procedures for including one or more RFC 7686 ".onion" special-use Domain Names in a Certificate.
+
+1. The Domain Name MUST contain at least two labels, where the right-most label is "onion", and the label immediately preceding the right-most "onion" label is a valid Version 3 Onion Address, as defined in section 6 of the Tor Rendezvous Specification - Version 3 located at https://spec.torproject.org/rend-spec-v3.
+2. The CA MUST verify the Applicant’s control over the .onion Domain Name using at least one of the methods listed below:
+    a. The CA MAY verify the Applicant’s control over the .onion service by using method 3.2.2.4.6, Agreed‐Upon Change to Website. If this method is replaced by a newer version(s) of Agreed-Upon Change to Website, the timelines for use of new and existing version of this method that are defined in section 3.2.2.4 SHALL apply.	
+    b. The CA MAY verify the Applicant's control over the .onion service by having the Applicant provide a Certificate Request signed using the .onion public key if the Attributes section of the certificationRequestInfo contains:
+        (i) A caSigningNonce attribute that contains a Random Value that is generated by the CA; and
+        (ii) An applicantSigningNonce attribute that contains a single value with at least 64-bits of entropy that is generated by the Applicant.
+	The signing nonce attributes have the following format:
+```
```

Comment:
Surely this doesn't render correctly, since you're trying to markdown in an escape sequence?

Reply:
Curiously enough, it does! (And don't call me Shirley. :) ) Or at least, I think so--see attached screenshot for how this renders out in PDF form, and the Word doc looks much the same. I definitely had to play around with the spacing of the numbering sequence to get it to come out right.

![br_sc26_sc27](https://user-images.githubusercontent.com/1791326/76446419-698bb400-639d-11ea-94ce-e8a54153d4b3.png)

Reply:
@castillar That's the /wrong/ output, although you're correct it's what @wthayer drafted in SC27, because that was copied over from EVG.md

You can see the 'expected' output in Appendix F of the current EV Guidelines, https://cabforum.org/wp-content/uploads/CA-Browser-Forum-EV-Guidelines-v1.7.1.pdf

When @dzacharo tried to fix the formatting in https://github.com/cabforum/documents/commit/ab17c2f9e3748cd54c7191fcec1fb8b8b206459d , he code quoted the sections, but this broke the markdown from when @benwilsonusa originally landed it in https://github.com/cabforum/documents/commit/ded42886c0569b181063439f804f9fa02378829d

That is, the correct approach for this section is either:
A) Use code-quoting (\`\`\`) for this section, and use whitespace to indicate correctness
B) Do not use code-quoting, and use tables to get the correct alignment

I think A is the correct answer here.

For a comparison with that is expected, beyond pointing to the EVGs that it's lifted from, [RFC 5912](https://tools.ietf.org/html/rfc5912) is relevant.

So, I know I suggested on the list we might be able to punt on SC27, but I think this matter is severe enough that we should fix it here, and have this replace the SC27 text.

Reply:
Whoof. Yeah, so this is one of those areas that strict Markdown doesn't handle well, partly because we're interleaving lists with non-list stuff, and partly because it's just a very specific layout. Thanks for the reference, though: that's exactly what I needed. I went with option A and then tried to adjust the spacing to make things come out correctly below the code-quote block. Does this look better?
![sc26_sc27_appC](https://user-images.githubusercontent.com/1791326/76452643-d4da8380-63a7-11ea-88d4-351089facba4.png)

Reply:
Yup. That looks right!